erpnext/hr/doctype/holiday_list/holiday_list.py

```diff
@@ -60,40 +84,20 @@ def clear_table(self):
 
 	def update_default_holiday_list(self):
 		frappe.db.sql("""update `tabHoliday List` set is_default = 0
-			where is_default = 1 and fiscal_year = %s""", (self.fiscal_year,))
+			where is_default = 1""")
```

Comment:
- [ ] Remove `is_default`
- [ ] Write a patch to set "Default Holiday List" for a Company based on `is_default`